BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationHelper.m

```diff
@@ -59,13 +59,13 @@ + (void)deleteFileFromLocal:(NSString *) fileName {
 
 + (CGFloat)convertPointsHeightToPercentage:(float) height {
     CGFloat presentationAreaHeight = [self getPresentationAreaHeight];
-    CGFloat heightInPercentage = (CGFloat) ceil(((height/presentationAreaHeight) * 100.0f));
+    CGFloat heightInPercentage = (CGFloat) (((height/presentationAreaHeight) * 100.0f));
```

Comment:
I believe this is to support float values in percentage

Reply:
Yes, so while converting percentage height back to points, it will give exact points.